assets/css/desktop.css

```diff
@@ -0,0 +1,464 @@
+  display: flex;
+  flex-direction: column;
+  align-self: center;
+  justify-items: center;
+  margin-left: 260px;
+  margin-right: 260px;
+}
+
+.intro-text {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 60px;
+  font-family: 'Poppins', sans-serif;
+  color: #091e42;
+  margin: 12px 0;
+}
+
+.intro-message {
+  font-family: 'Poppins', sans-serif;
+  font-size: 18px;
+  color: #344563;
+}
+
+.connect h3 {
+  color: #6070ff;
+}
+
+.social {
+  display: flex;
+  padding-left: 0;
+  list-style: none;
+  width: 200px;
+  margin: 0 25px 0 0;
+}
+
+.social-media {
+  font-size: 20px;
+  color: #505f79;
+  flex: 1;
+}
+
+.social-media a {
+  color: #505f79;
+}
+
+/*** Headline section End ***/
+
+/*** Works section **/
+
+.works-flex {
+  background: white;
+  margin: 142px;
+  border-radius: 8px;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+}
+
+.start {
+  order: 1;
+}
+
+.end {
+  order: 2;
+}
+
+.img-style {
+  display: none;
+}
+
+.img-style-desktop {
+  display: block;
+}
+
+.work-image {
+  align-self: center;
+  border-radius: 10px;
+  margin: 24px;
+}
+
+/* Text */
+
+.intro-text-works {
+  font-weight: 650;
+  padding-top: 10px;
+  font-size: 18px;
+  font-family: 'Poppins', sans-serif;
+  color: #344563;
+  margin: 0 0 0 0;
+}
+
+.titles-container {
+  display: flex;
+  flex-direction: row;
+  padding-left: 0;
+  justify-self: flex-start;
+  padding-top: 10px;
+  width: 100%;
+}
+
+.titles-item {
+  font-family: 'Poppins', sans-serif;
+  font-size: 16px;
+  color: lightslategray;
+  font-weight: 500;
+  list-style-image: url("../images/Counter.png");
+  flex: 2;
+}
+
+/* Text */
+
+.list-main {
+  list-style-type: none;
+  font-weight: 600;
+  font-family: 'Poppins', sans-serif;
+  font-size: 18px;
+  color: #344563;
+  padding-right: 25px;
+  flex: 1;
+  line-height: 24px;
+}
+
+.works-description {
+  padding-top: 10px;
+  margin-top: 0;
+  font-size: 15px;
+  font-weight: normal;
+  font-family: 'Poppins', sans-serif;
+  color: #344563;
+}
+
+.wcontainer {
+  margin: 15px;
+}
+
+.lang-container {
+  width: auto;
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  padding: 5px 5px 5px 0;
+  margin-left: 0;
+}
+
+.lang {
+  background: #ebebff;
+  border-radius: 8px;
+  font-size: 12px;
+  font-weight: 500;
+  padding: 4px 12px 4px;
+  color: #6070ff;
+  margin: 5px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+}
+
+/**** Works section end ****/
+
+/*** About section **/
+
+.aboutme {
+  height: 600px;
+}
+
+.about-headline {
+  margin-left: 142px;
+  margin-right: 142px;
+}
+
+.aboutme-wrapper {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  height: 100%;
+  padding-top: 100px;
+  background-color: white;
+  border-top-right-radius: 110px;
+  width: 100%;
+}
+
+.dflex {
+  background: #f7f7f99a;
+  border-radius: 10px;
+  list-style: none;
+  align-items: center;
+  font-size: medium;
+  font-weight: 500;
+  width: 122px;
+  height: 120px;
+  display: inline-block;
+  margin-left: 10px;
+}
```

Comment:
- [ ] Your design looks almost like the original one, to make it perfect, you should consider adding a `padding-top` as well as the right `font-size`.

Figma design:
![image](https://user-images.githubusercontent.com/66967274/144152787-1d3d952c-cec6-4882-aa77-5fa7c1601bb9.png)

Your design:
![image](https://user-images.githubusercontent.com/66967274/144153022-2ffacf57-f8be-4a09-9a8f-67d394e6c7c4.png)

Current typography for your element:
![image](https://user-images.githubusercontent.com/66967274/144152962-584e35c5-a612-45ec-8eea-c027c515f365.png)